umh-core/pkg/diagnosis/engine_test.go

```
@@ -0,0 +1,599 @@
+		Expect(readiness[0].Availability).To(Equal(Ready))
+		Expect(readiness[1].Signal).To(Equal("Q"))
+		Expect(readiness[1].Availability).To(Equal(Ready), "a capable quiet signal is Ready")
+		Expect(readiness[2].Signal).To(Equal("N"))
+		Expect(readiness[2].Availability).To(Equal(NoInstrument), "an incapable signal has a NoInstrument row, NOT an absent row")
+
+		Expect(fired).To(HaveLen(1), "only the over-mark signal fires")
+		Expect(fired[0].Identity.Signal).To(Equal("F"), "a Ready-but-quiet signal contributes nothing to the fired set")
+	})
+
+	It("should reset a fired latch the moment its window is AllAbsent when the signal declares release-on-absent, while a non-release signal holds until its own demote clock elapses", func() {
+		type s6 struct{ v float64 }
```

golangci-lint (2.12.2)

[error] field v is unused

(unused)